07_hello_friend/friend.rb

```diff
@@ -0,0 +1,5 @@
+class Friend
+  def greeting(who = nil)
```

Comment:
Use nouns for variable names.

```
The variable name should reveal the intention of the programmer. It should be obvious what the programmer intended that the variable is used for. Don’t refer to an object that models an employee using the variable name e. Call it what it is: an employee. Likewise, when referring to values such as integers use intention revealing name for example use int highestScore instead of int i.
```

https://readlearncode.com/code-and-stuff/clean-code-variables/